Assets/HoloToolkit/Input/Scripts/Focus/RayStep.cs

```diff
@@ -9,43 +9,43 @@ namespace HoloToolkit.Unity
     [Serializable]
     public struct RayStep
     {
-        public RayStep(Vector3 origin, Vector3 terminus)
+        public RayStep(Vector3 origin, Vector3 terminus) : this()
```

Comment:
This looks like a good change, but, to help me understand, what is this fixing in this case? I was under the impression ` : this()` was only needed if not all properties are set inside the constructor. In this case, all four are set.

Reply:
`this` is required _because_ we're setting the values in the constructor.

Idk, I just keep getting a compiler warning about it, and figured we should heed the warnings.

Reply:
Do you happen to have the warning? I can't seem to repro it.

Definitely agreed on fixing warnings though :)

Reply:
![image](https://user-images.githubusercontent.com/13334553/34496114-233a315e-efad-11e7-8cb3-8b9973ec0af7.png)

Reply:
Better safe than sorry. Though I can't seem to raise the warning either - I've tried 4.6, 3.5, 2.0 and 2.0 subset.

Reply:
Wow could not repro either :) Perhaps Stephen has some flags turned on in VS?